### DISCUSSION TITLE
What's the difference between containers/ and pages/?

### DISCUSSION THREAD

**Discussion by huy-vuong** [react-static/react-static#1528] [Q&A] [answered]
In both cases, I can use `useRouteData()` and they can be referred to in `static.config.js`. The only thing I can see differently about them right now is that `pages/` is for single pages without any parameterized URL, whereas `containers/` is slightly more generic and is just a top-level component.

I'm sure I'm missing some pretty fundamental concepts here though, so when would I use one over the other? Or what's the intended separation between the two? (I'm currently using just `containers/` and not touching `pages/` aside from `index.tsx` and `404.tsx`, but I'm sure that's a blatant misuse.)

I've looked through the following sources, but haven't found an answer:

- https://github.com/react-static/react-static/blob/master/docs/concepts.md
- https://github.com/react-static/react-static/blob/master/docs/config.md
- https://github.com/react-static/react-static/tree/master/docs/guides
- https://github.com/react-static/react-static/blob/master/docs/api.md

**Comment by SleeplessByte** [comment, marked as answer]
Neither is special. It was a standard copied from other systems, such as next/gatsby/those who use redux.

React-static does *nothing* with these folders.
